SEED-platform/seed
# Superuser improvements

#### What's this PR do?
- Superusers can always get to the admin page now, even if they don't belong to the current org they have selected
- Orgs you don't belong to have a light pink background in the org dropdown (this should only apply to superusers)
- The admin link does not appear for non-superusers now
- Improved data resolutions for the notes page to happen before page load

#### How should this be manually tested?
- As a superuser, remove yourself from an org, switch to that org, and try going to the admin page.  It should allow you now.  Also check that the background is pink in the org dropdown
- As a non-superuser, the `Admin` link should no longer be visible
